tests/test_issue421_cross_session_scope.py

```
@@ -0,0 +1,52 @@
+"""Regression tests for issue #421 cross-session visibility bugs."""
+
+import importlib
+import os
+import sqlite3
+
+
+def _reload_beam_with_cross_session(enabled: bool):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_reload_beam_with_cross_session`

(ANN202)